pkg/services/ngalert/models/alert_rule.go

```diff
@@ -46,6 +46,9 @@ const (
 const (
 	RuleUIDLabel      = "__alert_rule_uid__"
 	NamespaceUIDLabel = "__alert_rule_namespace_uid__"
+
+	DashboardUIDAnnotation = "__dashboardUid__"
```

Comment:
Annotations are actually a set of label, so technically this is a `DashboardUIDAnnotationLabelName` (because label = name + value) but that's too long of a name.